addons/bestja_offers/templates.xml

```diff
@@ -117,15 +131,6 @@
             </t>
         </template>
 
-        <template id="bestja_offers.duplicate_application" name="Ponowne zgłoszenie">
```

Comment:
Ups. I don't think so you meant to remove this template. Most likely you didn't pull our changes before sending yours. I created the template on Saturday, so I really don't want it removed ;)